src/CodeGenerator/Program.cs

```diff
@@ -360,52 +411,65 @@ static void Main(string[] args)
                 writer.WriteLine(string.Empty);
                 writer.PushBlock($"namespace {projectNamespace}");
                 writer.PushBlock($"public static unsafe partial class {classPrefix}");
-                foreach (FunctionDefinition fd in defs.Functions)
+                EmitFunctions(false);
+                writer.PopBlock();
+                writer.PopBlock();
+
+                if (useInternals)
                 {
-                    if (TypeInfo.SkippedFunctions.Contains(fd.Name)) { continue; }
+                    writer.PushBlock($"namespace {projectNamespace}{InternalNamespace}");
+                    writer.PushBlock($"public static unsafe partial class {classPrefix}");
+                    EmitFunctions(true);
+                    writer.PopBlock();
+                    writer.PopBlock();
+                }
```

Comment:
This runs the function overload processing logic twice, once for non-internals, to put things in the `ImGuiNET.ImGui` class, and then a second time for internals which go into the `ImGuiNET.Internal.ImGui` class.